scripts/report_scripts/index.Rmd

```diff
@@ -104,10 +116,11 @@ plots_line_locations[[3]]
 
 ## Virus strains per date and location
 
-On this map you can see where different virus strains and their
-relative frequencies have been detected at specific locations over
-time.  Use the slider to select a specific date or hit the Play button
-to display all snapshots successively.
+On this map you can see the relative frequencies of identified variants of concern (VOC) of SARS-CoV-2 and at specific 
+wastewater sampling locations over time. The diameter of the circles displays the variant frequency. Colors of overlapping variants 
+are adding up. This plot visualizes the development of local outbreaks and variant abundance.  
+
+Use the slider to select a specific date or hit the Play button to display all snapshots successively.
```

Comment:
Maybe add: By double-clicking on the variants displayed on the legend the chosen variants can be disabled.